### ISSUE TITLE
Mod `lightoverlay` (4.3) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'

### ISSUE THREAD

**Issue by Decme** [shedaniel/LightOverlay-Fabric#32]
Hi, I'm seeing this error on the game output window upon launch:
"Mod `lightoverlay` (4.3) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'"

Fabric continues to load other mods fine, but LightOverlay fails to load. I'm on 1.15.2

LightOverlay-4.3
Fabric loader 0.7.8 build 184

**Comment by shedaniel** [comment]
`Mod lightoverlay (4.3) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'` shouldn't crash the game, do you have a log?

**Comment by Decme** [comment]
It's not crashing, and sorry I was mistaken, LightOverlay actually does load & work (took me a long while to find the shortcut F7) - I'm just curious why the output window sends that error on load.

**Comment by Decme** [comment]
It's not crashing, and sorry I was mistaken, LightOverlay actually does load & work (took me a long while to find the shortcut F7) - I'm just curious why the output window sends that error on load.

edit: I just realized, although it doesn't crash the client, it doesn't actually load. The F7 function I was using only works on Fabric with Carpetmod. When I tried on a Paper server it didn't work, leading me to believe the F7 function I used was a Carpetmod feature and the clientside LightOverlay mod was in fact not working.
